### DISCUSSION TITLE
webhook for Telegram notifications ?

### DISCUSSION THREAD

**Discussion by ithakaa** [henrygd/beszel#237] [Support]
Hey all, does anyone have an example webhook for telegram notifications

**Comment by henrygd** [comment]
I don't use Telegram, but please check the Shoutrrr docs for the correct format if you haven't already:

https://containrrr.dev/shoutrrr/v0.8/services/telegram/

**Reply by ithakaa** [reply]
I've seen that, but it's missing the key parameter, so don't see how it will work

**Reply by wwng2333** [reply]
> I've seen that, but it's missing the key parameter, so don't see how it will work

You need a bot from @botFather at telegram.
